tests/extras/datasets/pandas/test_gbq_dataset.py

```diff
@@ -135,7 +135,6 @@ def test_str_representation(self, gbq_dataset, save_args, load_args):
 
     def test_save_load_data(self, gbq_dataset, dummy_dataframe, mocker):
```

Comment:
can you add a test where we you change `load_args["query"]` (see e.g above) and make sure that `read_gbq` is called with the right thing?